code/modules/projectiles/ammo_datums.dm

```diff
@@ -3699,9 +3702,66 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	added_spit_delay = 0
 	spit_cost = 100
 	damage = 40
-	smoke_strength = 1
-	reagent_transfer_amount = 10
+	smoke_strength = 0.4
+	reagent_transfer_amount = 5
```

Comment:
Note - the attack is still transferring roughly 8u-10u of neuro on a direct hit, most likely the neuro transfer is being applied twice somehow. Looking for feedback on how to correct this - until then, this value is reduced.